### PR TITLE
fix continue btn misalignment when AOE no sympotoms is selected

### PR DESCRIPTION
**Pull request by zdeveloper**
## Related Issue or Background Info

- fix for https://github.com/CDCgov/prime-simplereport/pull/2587#issuecomment-923030810

## Changes Proposed

- fix the alignment

## Screenshots / Demos

https://user-images.githubusercontent.com/4952042/134078948-38690cd8-342d-4b1d-bb4d-3a696826fd94.mp4

### UI
- [ ] Any changes to the UI/UX are approved by design 
- [ ] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [ ] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [ ] Database changes are submitted as a separate PR
  - [ ] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [ ] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [ ] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [ ] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [ ] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

## Cloud
- [ ] DevOps team has been notified if PR requires ops support
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

File: frontend/src/app/testQueue/AoEForm/AoEModalForm.tsx [hunk 1]
```diff
@@ -70,7 +67,6 @@ const AoEModalForm = (props: AoEModalProps) => {
         saveCallback={saveCallback}
         isModal={true}
         noValidation={true}
-        formRef={formRef}
```
Comment (nullflux) [inline frontend/src/app/testQueue/AoEForm/AoEModalForm.tsx:73]: **question (non-blocking):** Was this ref just entirely unused?

**Review by nullflux — APPROVED** [review]
LGTM, one small q!

File: frontend/src/app/testQueue/AoEForm/AoEModalForm.tsx (hunk 1 above)
Comment (zdeveloper) [reply frontend/src/app/testQueue/AoEForm/AoEModalForm.tsx:73]: it was assigned but never used

File: frontend/src/scss/PrimeStyles.scss [hunk 2]
```diff
@@ -1076,20 +1076,16 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
...
   background: color("white");
+  transform: translateX(-20px) translateY(20px);
 
   @include at-media(tablet) {
-    position: fixed;
-    width: unset;
-    right: unset;
-    padding: 0;
-    bottom: units(7);
+    transform: translateX(-20px);
```
Comment (nathancrtr) [inline frontend/src/scss/PrimeStyles.scss:1088]: Nice CSS-ing 😎

**Pull request merged**